src/Parallel/Main.hpp

```diff
@@ -97,9 +98,22 @@ Main<Metavariables>::Main(CkArgMsg* msg) noexcept
   namespace bpo = boost::program_options;
   try {
     bpo::options_description command_line_options;
-    command_line_options.add_options()
-        ("help,h", "Describe program options")
-        ("check-options", "Check input file options")
+    command_line_options.add_options()                 //
+        ("help,h", "Describe program options")         //
+        ("check-options", "Check input file options")  //
+        ("dump-source-tree-as", bpo::value<std::string>(),
+         "If specified then an gzip archive of the source tree is dumped "
+         "with the specified and the executable is terminated. The archive can "
```

Comment:
I think you are missing a word after "specified" ... perhaps "specified name" ??